samples/logging/application-insights/sample.md

```diff
@@ -1,7 +1,16 @@
 ---
 title: Report metrics to Azure Application Insights
+summary: Shows how to report NServiceBus metrics data to Azure Application Insights
 component: Metrics
-reviewed: 2017-07-29
+reviewed: 2017-10-06
+tags:
+- Azure
+- Metrics
+related:
+ - nservicebus/operations/metrics
+ - samples/azure
```

Comment:
Not sure if `samples/azure` is appropriate here. I would be nice if this sample could be listed under both `Logging and Metrics` and `Azure` - commented again after fixing the path

Reply:
Not sure how that will render. I think best course is to get this merged and see how it shows up and then adjust.